public/app/features/alerting/unified/components/rule-editor/FolderAndGroup.tsx

```diff
@@ -380,6 +386,16 @@ function EvaluationGroupCreationModal({
     setValue('evaluateEvery', interval, { shouldValidate: true });
   };
 
+  const modalTitle = isGrafanaRecordingRule
+    ? t(
+        'alerting.folderAndGroup.evaluation.modal.text.recording',
+        'Create a new evaluation group to use for this recording rule.'
+      )
+    : t(
+        'alerting.folderAndGroup.evaluation.modal.text.alerting',
+        'Create a new evaluation group to use for this alert rule.'
+      );
```

Comment:
(fullstops here as well)

Reply:
good catchhh

Reply:
where?

Reply:
🙃 No idea. I must be imagining things! Sorry

Reply:
hahha